From ded3b23309169f4370252d5c2572c6a584b21bc3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20Br=C3=BCns?= <stefan.bruens@rwth-aachen.de>
Date: Fri, 21 Jun 2019 00:03:35 +0200
Subject: [PATCH 6/7] Remove explicit initialization of default constructed
 members

Summary:
All four are of type QPointer<PackageKit::Transaction> and are thus
default initialized, no need to do it explicitly.

Reviewers: lukas, jgrulich

Differential Revision: https://phabricator.kde.org/D21975
---
 src/declarative/pkupdates.cpp | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/src/declarative/pkupdates.cpp b/src/declarative/pkupdates.cpp
index 2b4583d..e2aa79a 100644
--- a/src/declarative/pkupdates.cpp
+++ b/src/declarative/pkupdates.cpp
@@ -47,11 +47,7 @@ namespace
 } // namespace {
 
 PkUpdates::PkUpdates(QObject *parent) :
-    QObject(parent),
-    m_updatesTrans(Q_NULLPTR),
-    m_cacheTrans(Q_NULLPTR),
-    m_installTrans(Q_NULLPTR),
-    m_detailTrans(Q_NULLPTR)
+    QObject(parent)
 {
     setStatusMessage(i18n("Idle"));
 
-- 
2.21.0

